packages/server-wallet/src/models/adjudicator-status.ts

```diff
@@ -0,0 +1,88 @@
+import {getChannelMode} from '@statechannels/nitro-protocol';
+import {SignedState} from '@statechannels/wallet-core';
+import Knex from 'knex';
+import _ from 'lodash';
+import {Model} from 'objection';
+
+import {Bytes32, Uint48} from '../type-aliases';
+
+export type AdjudicatorStatus =
+  | {
+      channelMode: 'Finalized';
+      states: SignedState[];
+    }
+  | {channelMode: 'Challenge'; states: SignedState[]}
+  | {channelMode: 'Open'};
+
+interface RequiredColumns {
+  readonly channelId: Bytes32;
+  readonly finalizesAt: Uint48;
+  readonly blockNumber: Uint48;
+  readonly blockTimestamp: Uint48;
+}
+export class AdjudicatorStatusModel extends Model implements RequiredColumns {
+  readonly channelId!: Bytes32;
+  readonly finalizesAt!: Uint48;
+  readonly blockNumber!: Uint48;
+  readonly blockTimestamp!: Uint48;
+  readonly states!: SignedState[];
+  static tableName = 'adjudicator_status';
+  static get idColumn(): string[] {
+    return ['channelId'];
```

Comment:
Does this need to be an array?
```suggestion
    return 'channelId';
```